doc/code/qml_tape.rst

```diff
@@ -173,4 +173,4 @@ For more details and examples, please see the tape documentation.
 .. automodapi:: pennylane.tape
     :no-main-docstr:
     :include-all-objects:
-    :skip: enable_tape, disable_tape
+    :skip: QNode, qnode, enable_tape, disable_tape

```

Comment:
Because of all the mocking done by `qml.enable_tape()`, `qml.qnode`, `qml.QNode`, and `qml.CircuitGraph` all actually point to the `pennylane/tape` versions 🙂

Reply:
So I am skipping them from showing at the `tape` module level, as they will instead be displayed at the `qml` level